.travis.yml

```
@@ -13,6 +13,10 @@ before_install:
   - jabba install ${JDK:=adopt@~1.11-0}
   - jabba use ${JDK:=adopt@~1.11-0}
 
+env:
+  global:
+    - FLUSH=false
```

```suggestion
    - DELETE_CACHE=false
```

(pet peeve) I think `FLUSH` is overloaded.